src/authenticationservices.cs

```diff
@@ -1002,6 +1003,18 @@ enum ASAuthorizationSecurityKeyPublicKeyCredentialDescriptorTransport {
 		Bluetooth,
 	}
 
+	[Mac (12,0), iOS (15,0), MacCatalyst (15,0), NoWatch, NoTV]
+	[Native]
+	enum ASCOSEAlgorithmIdentifier : long {
+		Es256 = -7,
+	}
+
+	[Mac (12,0), iOS (15,0), MacCatalyst (15,0), NoWatch, NoTV]
+	[Native]
+	enum ASCOSEEllipticCurveIdentifier : long {
+		P256 = 1,
+	}
```

Comment:
^ comment it
why ? it's not yet used in any API and it could possibly change before it's used (and that would be a breaking change for us)